src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.TeamFoundation.SourceControl.WebApi;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using NuGet.Versioning;
```

Comment:
why do these files only contain using changes? is that enough to just change what things bind to in the existing code and it all just works?

Reply:
Yup. Practically a drop in replacement.